### PR TITLE
Gen2a

### PR DESCRIPTION
**Pull request by alanvgreen**
Next set of changes for gen2 accelerator. Moving toward being able to build a test that integrates a systolic array and post processor.

### CODE REVIEW

File: proj/hps_accel/gateware/gen2/sysarray.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        accumulator may be sufficient, and will make more efficient
+        use of the FPGA fabric.
+
+    Attributes
+    ----------
+
+    a_inputs: [Signal(n * a_shape.width)], in
+      List of words forming the A inputs
+
+    b_inputs: [Signal(32)], in
```
Comment (danc86) [inline proj/hps_accel/gateware/gen2/sysarray.py:61]: I guess this is actually a list of `Signal(n * b_shape.width)` which in our case is `Signal(32)`, right?

**Review by danc86 — APPROVED** [review]
Exciting

File: proj/hps_accel/gateware/gen2/sysarray.py (hunk 1 above)
Comment (tcal-x) [reply proj/hps_accel/gateware/gen2/sysarray.py:61]: @alanvgreen why the difference between how `a_inputs` and `b_inputs` are declared?

Comment (alanvgreen) [reply proj/hps_accel/gateware/gen2/sysarray.py:61]: No good reason. Fixed.

Comment (tcal-x) [reply proj/hps_accel/gateware/gen2/sysarray.py:61]: @alanvgreen did you push your change, or am I misunderstanding something?

Comment (alanvgreen) [reply proj/hps_accel/gateware/gen2/sysarray.py]: fixed now

**Pull request merged**